app/views/organizations/home/index.html.erb

```diff
@@ -3,7 +3,8 @@
     <div class="row d-flex align-items-center">
       <div class="col-xxl-5  col-xl-6 col-lg-6 col-12">
         <div>
-          <h1 class="display-2 fw-bold mb-3">Rescue pets in <%= Current.organization.city %> at <u class="text-warning"><span class="text-primary"> <%= Current.organization.name %></span></u></h1>
+          <%# Where should I get this city information from? Current.user is nil (page can be accesses without login) and Current.organization doesnt have location yet (should I link it on this PR?) %>
```

Comment:
How should I proceed here?

Reply:
Just saw [213](https://github.com/rubyforgood/pet-rescue/issues/213), so maybe re-introduce this information when that gets done (?)

Reply:
Yes for now you can comment that out!

Reply:
done!